pkg/sql/logictest/testdata/logic_test/timetz

```diff
@@ -670,3 +670,8 @@ SELECT a FROM regression_44774 ORDER BY a
 
 statement ok
 DROP TABLE regression_44774
+
+query T
+SELECT '20:01:02+03:04:05' AS regression_65066
```

Comment:
does this test catch it? i thought the issue was in the CLI so we'd need something in cli_test.go?

(when i connect to crdb using psql, this query works fine on master. do logic tests use the cli formatting?)